src/Bicep.Core/Semantics/Namespaces/NamespaceResolver.cs

```diff
@@ -81,6 +81,12 @@ public IEnumerable<Symbol> ResolveUnqualifiedFunction(IdentifierSyntax identifie
             }
         }
 
+        public FunctionSymbol? GetKnownFunction(string functionName)
```

Comment:
Although there is no current overlap between namespace functions, it would be good to be precise here. What about the following, and checking `.Any()` in the place where it is used?

```suggestion
        public IEnumerable<FunctionSymbol> GetKnownFunctions(string functionName)
            => this.namespaceTypes.Values
                .Select(type => type.MethodResolver.TryGetFunctionSymbol(functionName))
                .OfType<FunctionSymbol>();
```